main.py

```diff
@@ -58,7 +61,7 @@ async def runner():
 
         await asyncio.gather(client.start(token=token, reconnect=True),
                              console.start(get_console_input),
-                             web_console.start())
+                             web_console.start(), API.start("localhost", 5050))
```

Comment:
For Docker container reasons, these can't be hardcoded and need to be configurable by cli args:

- [ ]  use `hostname` parameter instead of `"localhost"`
- [ ] add a parameter for API port instead of hard-coding it in.
- [ ] add a CLI arg for setting the API port.